### DISCUSSION TITLE
unable to download anything

### DISCUSSION THREAD

**Discussion by Gravitys34** [Avnsx/fansly#142] [Q&A]
So i did subscribe to a creator on fansly, i'm following the account and i properly (at least it seems) configure the app. 

here is the message i'm getting : 
 Info | 16:18 || Reading config.ini file ...

 info | 16:18 || Username validation successful!
 Info | 16:18 || Using user-agent: 'Mozilla/5.0 (Windows NT 10.0 [...] cko) Chrome/116.0.0.0 Safari/537.36'
 Info | 16:18 || Open download folder when finished, is set to: 'True'
 Info | 16:18 || Downloading files marked as preview, is set to: 'True'

 WARNING | 16:18 || Previews downloading is enabled; repetitive and/or emoji spammed media might be downloaded!
 Info | 16:18 || Targeted creator: '_changed_'

 Info | 16:18 || Initiating Messages procedure. Standby for results.
 WARNING | 16:18 || Could not find a chat history with _changed_; skipping messages download ..

 Info | 16:18 || Executing Timeline functionality. Anticipate remarkable outcomes!

 Info | 16:18 || Inspecting most recent Timeline cursor
 Info | 16:18 || Amount of Media in current cursor: 0 (scrapable: 0)

 WARNING | 16:18 || Low amount of Pictures scraped. Creators total Pictures: 0 | Downloaded: 0

 WARNING | 16:18 || Low amount of Videos scraped. Creators total Videos: 0 | Downloaded: 0
                    Follow the creator; to be able to scrape more media!
                    Subscribe to the creator; if you would like to get the entire content.

╔═
  Finished Normal type, download of 0 pictures & 0 videos! Declined duplicates: 0
  Saved content in directory: 'D:\temp\Fansly_Downloader_0.4.1-Windows\_changed__fansly'
  ✶ Please leave a Star on the GitHub Repository, if you are satisfied! ✶

**Comment by Avnsx** [comment]
```
WARNING | 16:18 || Low amount of Pictures scraped. Creators total Pictures: 0 | Downloaded: 0

WARNING | 16:18 || Low amount of Videos scraped. Creators total Videos: 0 | Downloaded: 0
```
It fetches 0 available content from the creator fansly in first place.

You might need to remove the auth token from the config.ini file and start fansly downloader again, so that it helps you connect itself to the correct account, which you are actually subscribed / following with.

If you still have the same issue, try using a VPN before starting fansly downloader or use Cloudflare's WARP available under https://1.1.1.1/

**Reply by Gravitys34** [reply]
so i did few tests : i removed the token from config.ini and restarted fansly, i also connected through airvpn.
i'm still having an issue, i have now dowloaded couple of pics that have been sent to me by the creator, but nothing else, i was expecting the tool to download pics and video from the home page, am i wrong ?
Thanks for you help

**Reply by Avnsx** [reply]
No you're right.
When you set username for TargetedCreator in the config.ini file and leave download_mode on "Normal" or "Timeline" it should just download all the content.
Not sure how to help you.

**Comment by testpillow** [comment]


**Reply by Avnsx** [reply]
https://github.com/Avnsx/fansly-downloader/issues/148
